pkg/client/unversioned/client_test.go

```diff
@@ -85,8 +85,8 @@ func (c *testClient) Setup(t *testing.T) *testClient {
 			version = testapi.Default.Version()
```

Comment:
c.Version is never set (the field is only used on line 83 and 94). Remove c.Version?